cik8s-cluster.tf

```diff
@@ -143,6 +143,57 @@ module "cik8s" {
     # This list of worker pool is aimed at mixed spot instances type, to ensure that we always get the most available (e.g. the cheaper) spot size
     # as per https://aws.amazon.com/blogs/compute/cost-optimization-and-resilience-eks-with-spot-instances/
     # Pricing table for 2023: https://docs.google.com/spreadsheets/d/1_C0I0jE-X0e0vDcdKOFIWcnwpOqWC8RQ4YOCgXNnplY/edit?usp=sharing
+    spot_linux_arm64_4xlarge = {
+      # 4xlarge: Instances supporting 3 pods (limited to 4 vCPUs/16 Gb) each with 1 vCPU/1Gb margin
+      name          = "spot-linux-arm64-4xlarge"
+      capacity_type = "SPOT"
+      # Less than 5% eviction rate, cost below $0.07 per pod per hour
+      instance_types = [
+        "m7g.4xlarge" # 16 vCPUs/64 Gb RAM
+      ]
+      block_device_mappings = {
+        xvda = {
+          device_name = "/dev/xvda"
+          ebs = {
+            volume_size           = 90 # With 3 pods / machine, that can use ~30 Gb each at the same time (`emptyDir`)
```

Comment:
not sure where it's coming from but same as intel/amd instances with no volume on the VM